Consul/Interfaces/IOperatorEndpoint.cs

```diff
@@ -45,6 +45,8 @@ public interface IOperatorEndpoint
         Task<QueryResult<ConsulLicense>> GetConsulLicense(string datacenter = "", CancellationToken ct = default);
         Task<QueryResult<string[]>> SegmentList(QueryOptions q, CancellationToken cancellationToken = default);
         Task<QueryResult<string[]>> SegmentList(CancellationToken cancellationToken = default);
+        Task<WriteResult<string>> CreateArea(Area area, WriteOptions q, CancellationToken ct = default);
```

Comment:
The result type should be `Task<WriteResult<AreaResponse>>`

Reply:
> The result type should be `Task<WriteResult<AreaResponse>>`

When I made that comment I assumed that `AreaResponse` will contain only an `ID` field.